README.md

```diff
@@ -153,29 +211,37 @@ Now that you have created a machine learning model, you can utilize the OpenScal
 
```

Comment:
On the previous line - the link to configure open scale dashboard is broken

Reply:
fixed.